web/src/routes/LicensesPage.tsx

```diff
@@ -45,11 +26,26 @@ const LicensesPage = (): ReactElement => {
     <LicenseItem name={item.name} license={item.licenses} version={item.version} onPress={item.licenseUrl} />
   )
 
+  const languageChangePaths = languages.map(({ code, name }) => {
+    const disclaimerPath = pathnameFromRouteInformation({ route: LICENSES_ROUTE, cityCode, languageCode: code })
+    return { path: disclaimerPath, name, code }
+  })
+
+  const locationLayoutParams = {
+    cityModel,
+    viewportSmall,
+    languageChangePaths,
+    feedbackTargetInformation: null,
+    route: LICENSES_ROUTE,
+    languageCode,
+    pathname,
+  }
+
   return (
-    <Layout>
-      <Caption title={t('openSourceLicenses')} />
-      <List items={licenses ?? []} renderItem={renderItem} noItemsMessage='' />
-    </Layout>
+    <LocationLayout isLoading={false} {...locationLayoutParams}>
```

Comment:
This should not be a location layout route since the licenses are not specific to one city/language. This breaks navigation for the link in the general footer e.g. in the city selection:
![image](https://user-images.githubusercontent.com/29095487/188271893-159d5e76-72a3-4574-90cf-d3b627660fcb.png)